### PR TITLE
Avoid raising dependency exception if rasa x is not installed

### LINKED ISSUE #6921: ModuleNotFoundError: No module named 'rasax'
Sentry Issue: [RASA-OPEN-SOURCE-W](https://sentry.io/organizations/rasahq/issues/1899971952/?referrer=github_integration)

```
ModuleNotFoundError: No module named 'rasax'
(1 additional frame(s) were not displayed)
...
  File "rasa\__main__.py", line 116, in main
    cmdline_arguments.func(cmdline_arguments)
  File "rasa\cli\x.py", line 338, in rasa_x
    run_locally(args)
  File "rasa\cli\x.py", line 426, in run_locally
    from rasax.community import local  # pytype: disable=import-error
```

### PR DESCRIPTION
**Pull request by tmbo**
fixes #6921

### CODE REVIEW

File: rasa/cli/x.py [hunk 1]
```diff
@@ -422,8 +422,18 @@ def _get_credentials_and_endpoints_paths(
...
-    # noinspection PyUnresolvedReferences
-    from rasax.community import local  # pytype: disable=import-error
+    try:
+        # noinspection PyUnresolvedReferences
+        from rasax.community import local  # pytype: disable=import-error
+    except ModuleNotFoundError:
+        from rasa.nlu.components import MissingDependencyException
+
+        raise MissingDependencyException(
+            f"Rasa X does not seem to be installed, but is needed for this CLI command."
```
Comment (tmbo) [inline rasa/cli/x.py:432]:
```suggestion
            f"Rasa X does not seem to be installed, but it is needed for this CLI command."
```

File: rasa/cli/x.py [hunk 2]
```diff
@@ -422,8 +422,18 @@ def _get_credentials_and_endpoints_paths(
 
 
 def run_locally(args: argparse.Namespace):
-    # noinspection PyUnresolvedReferences
-    from rasax.community import local  # pytype: disable=import-error
+    try:
+        # noinspection PyUnresolvedReferences
+        from rasax.community import local  # pytype: disable=import-error
+    except ModuleNotFoundError:
+        from rasa.nlu.components import MissingDependencyException
```
Comment (m-vdb) [inline rasa/cli/x.py:429]: can we move the exception definition to `rasa.exceptions`? feels weird to import from `rasa.nlu`. And I'd move that import to the top level at the same time.

**Review by m-vdb — APPROVED** [review]
One minor comment, other than that ✅

File: rasa/cli/x.py (hunk 2 above)
Comment (tmbo) [reply rasa/cli/x.py]: good point, done :+1:

**Pull request merged**